pkg/gardenlet/apis/config/types.go

```diff
@@ -270,6 +272,17 @@ type ShootCareControllerConfiguration struct {
 	ConditionThresholds []ConditionThreshold
 }
 
+// SeedCareControllerConfiguration defines the configuration of the SeedCare
+// controller.
+type SeedCareControllerConfiguration struct {
+	// SyncPeriod is the duration how often the existing resources are reconciled (how
+	// often the health check of Seed clusters is performed (only if no operation is
+	// already running on them).
```

Comment:
What do you mean by "only if no operation is already running on them"?

Reply:
Nothing, I think it is copied from another similar property